src/interactors/users/v1.ts

```diff
@@ -0,0 +1,16 @@
+import {User, UserFilters} from "../../types";
```

Comment:
Please use Prettier for formatting before submitting PR.

Reply:
@checkmatez we'll call you back 🙂